docs/adr/2021-06-01-patch-release-process.md

```diff
@@ -0,0 +1,45 @@
+# Make two releases a year. One major, one patch
+
+* Status: accepted
+* Deciders: @handrews @jdesrosiers @relequestual @gregsdennis @karenetheridge
+* Date: 2021-06-01
+
+Technical Story: https://github.com/json-schema-org/community/discussions/7
+
+## Context and Problem Statement
+
+JSON Schema has no formalised release process. Currently, we kind of release when we feel there's enough to bundle into a release, with a vague notion of IETF drafts expiring after 6 months.
+
+We want to have a reliable release cadence, to allow us to set pace, but also so others know what to expect.
+
+## Decision Drivers <!-- optional -->
+
+* We have occasionally found small fixes or clarifications would be helpful
+* We want to respond to feedback without having to wait a year due to slow processes
+
+## Considered Options
+
+We didn't formally consider any other options. No alternative proposals were offered.
+
+## Decision Outcome
+
+Chosen option: "[option 1]", because [justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force force | … | comes out best (see below)].
```

Comment:
Not sure what's going on here. Maybe scaffolding that was intended to be removed?

Reply:
Correct. I'll make adjustments.